homeassistant/components/dreo/translations/en.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Integrations should only provide translation strings (`strings.json` file).
Please remove this file.